lib/scrollable_clean_calendar.dart

```diff
@@ -121,13 +117,13 @@ class _ScrollableCleanCalendarState extends State<ScrollableCleanCalendar> {
   Widget build(BuildContext context) {
     return ListView.separated(
       controller: widget.scrollController,
-      padding: widget.padding ??
-          const EdgeInsets.symmetric(horizontal: 16, vertical: 32),
+      padding: widget.padding ?? const EdgeInsets.symmetric(horizontal: 16, vertical: 32),
       // cacheExtent:
       //     (MediaQuery.of(context).size.width / DateTime.daysPerWeek) * 6,
-      separatorBuilder: (_, __) =>
-          SizedBox(height: widget.spaceBetweenCalendars),
+      separatorBuilder: (_, __) => SizedBox(height: widget.spaceBetweenCalendars),
       itemCount: widget.calendarController.months.length,
+      reverse: true,
+      physics: const ClampingScrollPhysics(),
```

Comment:
Same comment above. I believe that we can add a parameter to the `physics` property. WDTY?